src-java/connecteddevices-topology/connecteddevices-storm-topology/src/test/java/org/openkilda/wfm/topology/connecteddevices/service/PacketServiceTest.java

```diff
@@ -510,4 +510,14 @@ private ArpInfoData createArpInfoData() {
         return new ArpInfoData(SWITCH_ID_1, PORT_NUMBER_1, newArrayList(VLAN_1), ARP_INPUT_PRE_DROP_COOKIE,
                 MAC_ADDRESS_1, IP_ADDRESS_1);
     }
+
+    private String createUniqueLldpIndex(LldpInfoData data, int vlan) {
+        return String.format("%s_%s_%s_%s_%s_%s_%s", data.getSwitchId(), data.getPortNumber(), LLDP,
+                vlan, data.getMacAddress(), data.getChassisId(), data.getPortId());
+    }
+
+    private String createUniqueArpIndex(ArpInfoData data, int vlan) {
+        return String.format("%s_%s_%s_%s_%s_%s", data.getSwitchId(), data.getPortNumber(), ARP,
+                vlan, data.getMacAddress(), data.getIpAddress());
+    }
```

Comment:
We have 3 places which creates such index. Maybe we should move it to some common place? For example SwitchConnectedDevice?

Reply:
Actually, in one of these 3 places, the method has a different signature and src-java/kilda-persistence-tinkerpop/src/test/java/org/openkilda/persistence/ferma/repositories/FermaSwitchConnectedDevicesRepositoryTest.java doesn't have a dependency on ArpInfoData/LldpInfoData. So, I updated the method usage similarly to `findFlowRelatedDataForOneSwitchFlow` method.

Reply:
Okay, What we have now:
1. `createUniqueLldpIndex` in `PacketService`
2. `createUniqueLldpIndex` in `FermaSwitchConnectedDevicesRepositoryTest`
3. `recalculateUniqueIndex` in `SwitchConnectedDeviceFrame`

each method does the same. If we will change one - we must change all. Also you a user will need to read devices in some separate - he will must to create another one method `createUniqueLldpIndex` in this separate module. 

What I'm proposing: 
1. Create a static method in SwitchConnectedDevice. something like `createUniqueLldpIndex(switcchid, port, .....`
2. Use this method everywhere

Reply:
Fixed. Thanks!